packages/lab/src/tabs-next/TabstripNext.css

```diff
@@ -0,0 +1,138 @@
+/* Component class applied to the root element */
+.saltTabstripNext {
+  --tabs-tabstrip-height: var(--saltTabs-tabstrip-height, var(--salt-size-stackable));
```

Comment:
size-stackable is now size-base + spacing-100 where spacing-100 addition is coming from the padding top and bottom. would be good to use the new tokens as stackable is being deprecated eventually - not sure if new sizes are in tabs spec yet though so can be done later

Reply:
I will check with Nav